airflow/configuration.py

```diff
@@ -216,6 +216,18 @@ def get(self, section, key, **kwargs):
                 "section/key [{section}/{key}] not found "
                 "in config".format(**locals()))
 
+    def safe_get(self, section, option, default):
```

Comment:
I understand that you are implementing these `safe_{get,getboolean}`, but it makes the configuration.py a bit hairy. Maybe for now just make sure that these elements are in the config file? This has been implemented to make sure that you don't silently miss any configuration sections/keys.